packages/core/src/SessionEventListener.ts

```diff
@@ -20,11 +20,42 @@
 //
 import type { EventEmitter } from "events";
 import type { EVENTS } from "./constant";
+import type { KeyPair } from "./authenticatedFetch/dpopUtils";
 
 export interface IHasSessionEventListener {
   events: ISessionEventListener;
 }
 
+/**
+ * A set of tokens to be passed to the client
+ */
+export type SessionTokenSet = {
+  /**
+   * JWT-serialized access token
+   */
+  access_token: string;
```

Comment:
```suggestion
  accessToken: string;
```
The snake casing is aligned with the specification, but we'll want to use camel case for our type. I'm surprised the linter didn't complain, we have to add linting exceptions for such names in other places. Can you check the linter runs appropriately on the core package?